release/nightly_tests/dataset/read_parquet_benchmark.py

```diff
@@ -6,34 +6,36 @@
 
 import shutil
 import tempfile
+from typing import Optional
 
 
 def read_parquet(
     root: str,
-    parallelism: int = -1,
+    override_num_blocks: Optional[int] = None,
     use_threads: bool = False,
     filter=None,
     columns=None,
 ) -> Dataset:
     return ray.data.read_parquet(
         paths=root,
-        override_num_blocks=parallelism,
+        override_num_blocks=override_num_blocks,
         use_threads=use_threads,
         filter=filter,
         columns=columns,
-    ).materialize()
```

Comment:
need to remove the materialize, since the benchmark function `run_materialize_ds` runs the timer when materializing.